mlflow/store/db/utils.py

```diff
@@ -55,7 +55,7 @@
 
 _logger = logging.getLogger(__name__)
 
-MAX_RETRY_COUNT = 15
+MAX_RETRY_COUNT = 10
```

Comment:
`MAX_RETRY_COUNT=15` results in sleeping for almost an hour in total, which is too long.